packages/web/src/scss/components/Radio/_Radio.scss

```diff
@@ -70,54 +74,57 @@ $_field-name: 'Radio';
     @include form-fields-tools.label-required();
 }
 
-.Radio--disabled {
-    @include form-fields-tools.inline-field-root-disabled();
-}
+.Radio--item {
+    @include form-fields-tools.item();
 
-.Radio--disabled .Radio__label {
-    @include form-fields-tools.label-disabled();
+    display: flex;
```

Comment:
Maybe we could remove this here and in `Checkbox`, there is already `width: 100%` in the mixin above.